dashboard/legacy/middleware/helpers/auth_helpers.rb

```diff
@@ -62,14 +62,14 @@ def has_permission?(permission)
 #          Note: NOT always true for admins.
 def owns_section?(section_id)
   return false unless section_id && current_user_id
-  DASHBOARD_DB[:sections].where(id: section_id, user_id: current_user_id).any?
+  DASHBOARD_DB[:sections_instructed].where(id: section_id, user_id: current_user_id).any?
```

Comment:
Can you verify there's good test coverage for the two changes in this file? I have less confidence about them being no-ops than the rest. 

Also this particular one seems to still be looking at section ownership even after swapping out the terminology. Might be good to create a follow-up task to investigate where this is used and whether it might be better to change this method to `instructs_section?`.

Reply:
I removed these to deal with in a future PR